features/view_specs/view_spec.feature

```diff
@@ -1,6 +1,55 @@
 Feature: view spec
 
-  View specs live in spec/views and render view templates in isolation.
+  View specs should be placed in `spec/views`,
+  and verify the content of view templates in isolation
+  (that is, without invoking a controller).
```

Comment:
Theres no verification that takes place.

Reply:
Alright; is 'test' acceptable instead?

Reply:
As in:

> Use them to test the content of view templates in isolation (that is, without invoking a controller).